tests/wpt/web-platform-tests/XMLHttpRequest/formdata-get.htm

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<html lang=en>
+<meta charset=utf-8>
+<title>FormData: get and getAll</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+    <link rel="help" href="https://xhr.spec.whatwg.org/#dom-formdata-get" />
+    <link rel="help" href="https://xhr.spec.whatwg.org/#dom-formdata-getall" />
+<div id="log"></div>
+<form id="form">
+    <input type="hidden" name="key" value="value">
+</form>
+<script>
+    test(function() {
+        assert_equals(create_formdata(['key', 'value1'], ['key', 'value2']).get('key'), "value1");
+    }, 'testFormDataGet');
+    test(function() {
+        assert_equals(new FormData(document.getElementById('form')).get('key'), "value");
+    }, 'testFormDataGetFromForm');
+    test(function() {
+        assert_equals(create_formdata(['key', 'value1'], ['key', 'value2']).get('nil'), null);
+    }, 'testFormDataGetNull1');
+    test(function() {
+        assert_equals(create_formdata().get('key'), null);
+    }, 'testFormDataGetNull2');
+    test(function() {
+        assert_array_equals(create_formdata(['key', 'value1'], ['key', 'value2']).getAll('key'), ["value1", "value2"]);
+    }, 'testFormDataGetAll');
+    test(function() {
+        assert_array_equals(create_formdata(['key', 'value1'], ['key', 'value2']).getAll('nil'), []);
+    }, 'testFormDataGetAllEmpty1');
+    test(function() {
+        assert_array_equals(create_formdata().getAll('key'), []);
+    }, 'testFormDataGetAllEmpty2');
+
```

Comment:
Since we already have a lot of the other formdata methods implemented, could we add tests for those as well? Including tests that thrash the dataset by repeated delete/replace/sets. (Add tests for all formdata methods, even ones we don't support, ensure that they pass on firefox)